docs/src/reference/asciidoc/en/deep-dive.adoc

```diff
@@ -1,5 +1,108 @@
 == Deep-Dive
 
+=== Representation of an authenticator
+
+WebAuthn4j provides `Authenticator` interface as a representation of an authenticator.
+
+On registering the authenticator, you need to persist its representation by creating the instance implementing
+`Authenticator` interface in your application manner because it is used afterwards on authentication verification.
+It might be better to use credentialId as a search key for this persisted instance.
+
+You can freely enhance the class implementing `Authenticator` interface in order to meet your application's requirements.
+For example, you can add the field for your application's user to identify the authenticator.
+
+=== Authenticator serialization and deserialization
+
+While it is application's responsibility to serialize `Authenticator` instance at registration, WebAuthn4J provides
+an utility class to serialize or deserialize members of `Authenticator` class. Please use them for implementing persistence in your application.
+
+==== attestedCredentialData
+
+`AttestedCredentialDataConverter` converts from `AttestedCredentialData` to `byte[]` and vice versa.
+If you would like to persist as `String`, use `Base64UrlUtil` to convert from `byte[]` to base64url `String`.
+
+[source,java]
+----
+AttestedCredentialDataConverter attestedCredentialDataConverter = new AttestedCredentialDataConverter(objectConverter);
+
+// serialize
+byte[] serialized = attestedCredentialDataConverter.convert(attestedCredentialData);
+// deserialize
+AttestedCredentialDataConverter deserialized = attestedCredentialDataConverter.convert(serialized);
```

Comment:
Hi @ynojima, there is a typo here, the static type should be ```AttestedCredentialData```.

Reply:
Thank you for pointing out! 
Fixed in https://github.com/webauthn4j/webauthn4j/pull/603